<launch>
    <node name="detection_node" pkg="im_detector" type="detection_node.py">
        <param name="image_topic" value="/color/image_raw" />
        <param name="camera_info_topic" value="/color/camera_info" />
        <param name="detection_topic" value="/im_detector/objects" />
    </node>
</launch>
